docker-compose.yml

```diff
@@ -1,8 +1,7 @@
 server:
   build: .
   volumes:
-    - .:/app
-    - /app/node_modules
+    - ./conf.js:/app/conf.js
```

Comment:
i still think it should be the other way around, have docker-compose.yml as your default dev environment and docker-compose.live.yml as your live trading one. @DeviaVir what do you think?